tests/wpt/web-platform-tests/dom/nodes/NodeList-Iterable.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>NodeList Iterable Test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+    <p>wake up</p>
+<script>
+    var paragraphs;
+    setup(function() {
+        paragraphs = document.querySelectorAll('p');
```

Comment:
I don't think `querySelectorAll` is implemented on the document, and I'm not sure whether there's work going towards implementing CSS Selectors DOM interface.

Pinging @SimonSapin to see if we'd want to work on that.